frontend/src/bundles/common/components/sort/sort.tsx

```diff
@@ -0,0 +1,55 @@
+import clsx from 'clsx';
+import {
+    type ButtonHTMLAttributes,
+    type MouseEventHandler,
+    type ReactNode,
+} from 'react';
+
+import { IconName } from '../../enums/enums';
+import { Icon } from '../components';
+import styles from './styles.module.scss';
+
+type SortType = 'asc' | 'desc' | null;
+
+interface Properties extends ButtonHTMLAttributes<HTMLButtonElement> {
+    sortType?: SortType;
```

Comment:
Set default value to this prop and add logic of changing sort type to this component

Reply:
All fixes was made